docs/analytics.md

```diff
@@ -0,0 +1,132 @@
+
+<i>Last updated: Feb 25, 2019</i>
+
+<br />
+
+# Firefox Monitor Analytics Plan
+
+This is the Analytics plan for Firefox Monitor. It documents our use of Google Analytics and what we do with the information we collect.
+
+<br>
+
+## Analysis
+**Firefox Monitor uses Google Analytics to collect and organize data. We do this to get a better understanding of what is working, and where we still have work to do.**
+
+
+**Captured data also helps provide answers to the following questions:**
+
+
+**Demographic:**
+
+>From which country does the majority of our traffic originate?
+
+>In which language does the majority of our traffic view the site?
+
+>Which browsers are most commonly used to access Firefox Monitor?
+
+>Which devices are most commonly used to access Firefox Monitor?
+
+<br>
+  
+**User Behavior:**
+
+>Do users scan their email?
+
+>Do users scan multiple email addresses?
+
+>What percentage of users sign up receive alerts from Firefox Monitor?
+
+>Are users more likely to sign up for alerts before or after scanning their email?
+
+
+<br />
+
+## Collection
+
+Events are reported using the [Google Analytics Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/).
+
+We collect data for the following events:
+
+- When the page loads (or reloads).
+`hitType` : pageView
+`page` : location.pathName
+<br>
+
+#### Forms:<br><br> `Email Scan Form` &nbsp; `Scan Another Email Form` &nbsp; `Sign Up Form` &nbsp;
```

Comment:
Line breaks in a heading seem odd.
Not sure if you want to convert to something like:

```md
### Forms:

#### `Email Scan Form` &nbsp; `Scan Another Email Form` &nbsp; `Sign Up Form`
```

---

### Forms:

#### `Email Scan Form` &nbsp; `Scan Another Email Form` &nbsp; `Sign Up Form`

---

This keeps your H1-H2-H3-H4 continuity and may be slightly more correct.
I can't really think of any other ways to add padding between the strings, unless you wanted to use an &amp;mdash; (&mdash;) or &amp;ndash; (&ndash;), or put them in an `<ol>` or `<ul>`, or even a `<table>`:

| `Email Scan Form` | `Scan Another Email Form` | `Sign Up Form` |
|-------------------|---------------------------|----------------|